packages/appstore/src/components/main-title-bar/total-asset.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Text, Popover } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import BalanceText from 'Components/elements/text/balance-text';
+import { useDelayedTotalAccountBalance, useCFDAccounts, usePlatformAccounts } from '@deriv/hooks';
+
+const TotalAsset = observer(() => {
+    const { traders_hub, client, common } = useStore();
+    const { selected_account_type, is_eu_user, no_CR_account, no_MF_account } = traders_hub;
+    const { default_currency } = client;
+    const { current_language } = common;
+    const { real: platform_real_accounts, demo: platform_demo_account } = usePlatformAccounts();
+    const { real: cfd_real_accounts, demo: cfd_demo_accounts } = useCFDAccounts();
+    const platform_real_balance = useDelayedTotalAccountBalance(platform_real_accounts, 60);
```

Comment:
please put milliseconds here.
saying this as most of the time measurements in javascript is milliseconds.
So basically if we want to have 60 seconds we can go with 1000 * 60 to be more readable.

Reply:
you're right. will update this